finance_manager/src/test/java/com/exadel/finance/manager/FinanceManagerApplicationTests.java

```diff
@@ -1,15 +1,11 @@
 package com.exadel.finance.manager;
 
-import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.ActiveProfiles;
 
 @SpringBootTest
 @ActiveProfiles("test")
 class FinanceManagerApplicationTests {
 
-    @Test
-    void contextLoads() {
```

Comment:
any specific reason to remove the basic test please?

Reply:
I'm sorry, I'll return it as it was